src/test/java/edu/harvard/iq/dataverse/api/InfoIT.java

```diff
@@ -81,6 +84,19 @@ public void testGetZipDownloadLimit() {
                 .body("data", notNullValue());
     }
 
+    @Test
+    public void testGetExportFormats() {
+        Response response = given().urlEncodingEnabled(false)
+                .get("/api/info/exportFormats");
+        response.prettyPrint();
+        response.then().assertThat().statusCode(OK.getStatusCode());
+
+        String expectedJson = UtilIT.getDatasetJson("src/test/resources/json/export-formats.json");
+        JsonObject expectedJsonObject = new Gson().fromJson(expectedJson, JsonObject.class);
```

Comment:
We are trying to use standard APIs rather than stuff like Gson.

Can you please try something like this from SignpostingIT?

```
JsonObject data = JsonUtil.getJsonObject(responseString);
```

Reply:
I just created https://github.com/julian-schneider/dataverse/pull/1 which does the following:

- Removes Gson, which we are trying to avoid using (see #6810).
- Introduces JSONAssert, to avoid testing on a simple string match. Instead it tests that the JSON docs are equivalent, even if the keys are out of order, etc.
- Pretty prints the "expected" JSON file to be readable.